File: main/java/com/marsrovers/location/PlateauFactory.java
package com.marsrovers.location;

public class PlateauFactory {

    private static final int DEFAULT_HORIZONTAL_SIZE = 10;
    private static final int DEFAULT_VERTICAL_SIZE = 10;

    public Plateau createDefaultPlateau() {
        return new Plateau(DEFAULT_HORIZONTAL_SIZE, DEFAULT_VERTICAL_SIZE);
    }

}

File: main/java/com/marsrovers/location/PlateauFactory.kt
package com.marsrovers.location

class PlateauFactory {

    fun createDefaultPlateau(): Plateau = Plateau(DEFAULT_HORIZONTAL_SIZE, DEFAULT_VERTICAL_SIZE)

    companion object {

        private val DEFAULT_HORIZONTAL_SIZE = 10
        private val DEFAULT_VERTICAL_SIZE = 10
    }

}
